components/CardFooter.vue

```diff
@@ -1,12 +1,16 @@
 <template>
   <footer class="card__snap-footer">
     <p class="url"><span class="label">URL:</span> {{ thisUrl }}</p>
-    <p class="date"><span class="label">Date:</span> <time :datetime="this.today">{{ this.today }}</time></p>
+    <p class="date"><span class="label">{{ $t('Date:', locale) }}</span> <time :datetime="this.today">{{ this.today }}</time></p>
```

Comment:
@rupl Should the : colon go into the translation string?
`{{ $t('Date:', locale) }}`
or
`{{ $t('Date', locale) }}:`

Reply:
It's a good question. In cases of single words or short phrases I'd tend to omit punctuation.

A counter-example would be an exclamation or question. Since we're translating to Spanish eventually and Spanish marks at the beginning and end of a sentence, then including makes more sense to me in that case.